### PR TITLE
Luxury ships

### PR DESCRIPTION
**Pull request by ph2000bis**
Due to compatibility problems in the branch, redid the pull request
Fixed issues from
Tested, the ships appear in shipyard under "luxury ships"
Problem with the `has "ships: Luxury Transport"` condition

### CODE REVIEW

**Comment by ph2000bis** [comment]
Why does this branch have compatibility issues ?

File: data/jobs.txt [hunk 1]
```diff
@@ -1573,3 +1573,104 @@ mission "Large Rush Delivery [3]"
...
 		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+		
+mission "Wealthy Executive Short [0]"
+	name "Transport a wealthy executive to <planet>"
+	job
+	repeat
+	deadline 0 1
+	passengers 1
+	description "Bring a wealthy and important executive to <destination> by <date>."
+	blocked "You cannot accept this mission, you have either run out of bunks or of luxury bunks for your rich and important travelers (the base number of bunks on your ship)"
```
Comment (egilburg) [inline data/jobs.txt]: text about bunks no longer relevant

**Comment by Gkramme** [comment]
The missions need to be higher paying.

**Comment by ph2000bis** [comment]
10000 + 300*((2 to 4)+1)*10 = 19000 to 25000 for a single passenger 2-4 jumps, isn't that high enough
In comparison, tourist bring you 
6000 + 150*((2-4)+1)*10
10500 to 13500 for the same distance and one passenger
Increased base by 10000 and 15000 for the longer runs

**Comment by Gkramme** [comment]
I mean, they are rich people. They should be paying you at least 250K for the SMALL missions. Bounty hunting is much more profitable. However, to counter this, they should become much rarer

**Comment by natangold** [comment]
@Gkramme These rich people are not *moving* to a new planet (or they'd be bringing tons of `cargo "luggage"`, so you'd be looking at 500k for a round-trip ticket. At that price point, a CEO could buy a Flivvers and a Sparrow and pay 200 credits a day for a chauffeur and a bodyguard. If a CEO expects she will need to make more than two of these trips ever, they would just buy their own Arrow (at 1.2m) and *never* have the player chauffeur them around again. I grok what you're saying - these rich jerks should be paying even more than some lot of bourgeois tourists, but you have to keep in mind that hiring the player is only one option if you're in the 1% (and is really… ugh… just so plebeian, honestly).

Consider also that, IRL, there is a market for some retirees who simply leave land behind and [permanently move aboard a cruise ship](http://money.usnews.com/money/blogs/on-retirement/2015/08/07/all-aboard-retire-on-a-ship) - because it can be [literally cheaper than living in a nursing home](http://www.snopes.com/travel/trap/retire.asp). For someone not quite rich enough to retire to Hestia, living aboard a Star Queen may well be the next best thing.

22k for 3 hops might be on the low side, but I think it's closer to equilibrium than having luxury transportation being more profitable than shooting pirates out of the sky. Perhaps a happy medium somewhere between the two?

**Comment by Gkramme** [comment]
Yea, maybe you could. You could also escort their arrows or flyvers for a lot more money

**Comment by asl97** [comment]
@ph2000bis it's because your patch is conflicting with commits after [Added the Wanderer territory to the map.](https://github.com/endless-sky/endless-sky/commit/403def1a596c2471657bfb94bcae5c2c090802a1)

https://github.com/endless-sky/endless-sky/commits/master/data/jobs.txt

**Comment by ph2000bis** [comment]
Fixed, the branch is now compatible for merging

**Pull request closed without merging**